AppTaskQueue/templates/task.py

```diff
@@ -69,12 +69,19 @@ def get_wait_time(retries, args):
       celery.control.revoke(QUEUE_NAME.request.id)
       return
 
+    # For tasks generated by mapreduce, or destined to be run by a module,
+    # the hostname may have a prefix that corresponds to a different subdomain.
+    # AppScale does not support that type of routing currently, so the main
+    # loadbalancer IP is used here for the execution of a task.
```

Comment:
Would it make sense for the AppServer to be responsible for providing the right hostname (or IP address)?